### PR TITLE
Added support for flutter web

### PR DESCRIPTION
**Pull request by abbasidaniyal**
I have removed dart:ui import as it was serving a very basic purpose.
By removing it and commenting few lines of code, this package is compatible with flutter web now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-[![Flutter Community: flutter_google_places](https://fluttercommunity.dev/_github/header/flutter_google_places)](https://github.com/fluttercommunity/community)
```
Comment (lejard-h) [inline README.md]: Why remove this ?

File: lib/src/flutter_google_places.dart [hunk 2]
```diff
@@ -1,7 +1,7 @@
 library flutter_google_places.src;
 
 import 'dart:async';
-import 'dart:io';
+// import 'dart:io' if () "" ;
```
Comment (lejard-h) [inline lib/src/flutter_google_places.dart]: you can simply remove it

File: lib/src/flutter_google_places.dart [hunk 3]
```diff
@@ -160,14 +160,13 @@ class _PlacesAutocompleteOverlayState extends PlacesAutocompleteState {
           Padding(padding: EdgeInsets.only(top: 48.0), child: body),
         ]));
 
-    if (Platform.isIOS) {
-      return Padding(padding: EdgeInsets.only(top: 8.0), child: container);
-    }
-    return container;
+    // if (Platform.isIOS) {
```
Comment (lejard-h) [inline lib/src/flutter_google_places.dart]: Test platform using `TargetPlatform.isIOS` from `package:flutter/foundation.dart`

File: lib/src/places_autocomplete_field.dart [hunk 4]
```diff
@@ -175,6 +175,8 @@ class _LocationAutocompleteFieldState extends State<PlacesAutocompleteField> {
     Prediction p = await _showAutocomplete();
 
     if (p == null) return;
+
```
Comment (lejard-h) [inline lib/src/places_autocomplete_field.dart]: revert unnecessary change

File: pubspec.yaml [hunk 5]
```diff
@@ -11,9 +11,9 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  rxdart: ^0.22.0
-  google_maps_webservice: ^0.0.14
-  http: ">=0.11.0 <1.0.0"
+  rxdart: 0.22.0
```
Comment (lejard-h) [inline pubspec.yaml]: Too restrictive
I don't think you need to change version here

**Comment by abbasidaniyal** [comment]
Sure! I'll just fix these up!

File: README.md (hunk 1 above)
Comment (abbasidaniyal) [reply README.md]: My bad. Fixed it!

File: lib/src/flutter_google_places.dart (hunk 2 above)
Comment (abbasidaniyal) [reply lib/src/flutter_google_places.dart]: Done.

File: lib/src/flutter_google_places.dart (hunk 3 above)
Comment (abbasidaniyal) [reply lib/src/flutter_google_places.dart]: Done

File: lib/src/places_autocomplete_field.dart (hunk 4 above)
Comment (abbasidaniyal) [reply lib/src/places_autocomplete_field.dart]: Sure!

File: pubspec.yaml (hunk 5 above)
Comment (abbasidaniyal) [reply pubspec.yaml]: Fixed it!

**Comment by abbasidaniyal** [comment]
I have made the changes as instructed by you. Do let me know if there's any other changes that I need to make.

**Comment by abbasidaniyal** [comment]
@lejard-h
